src/ShipBattleGame.java

```diff
@@ -0,0 +1,183 @@
+import java.util.Objects;
+import java.util.Random;
+import java.util.Scanner;
+
+public class ShipBattleGame {
+    static final Random RAND = new Random();
+    static final int N = 10;
+    static final Scanner SC = new Scanner(System.in);
+
+    public static void main(String[] args) {
+        game();
+    }
+
+    // This method add ship to the map and return str placements of ship
+    public static String[][] addShips(int[][] map) {
+        int[][] intShips = new int[6][];
+        String[][] strShips = new String[6][]; // for coordinates
+        int k = 0;
+        for (int i = 5; i > 1; i--) { // creating ships size of 5, 4, 3, 2
```

Comment:
![image](https://github.com/user-attachments/assets/e865174a-2bb2-467d-95aa-2594e172e5f6)

Reply:
as you see there should be different count for ships. In your case you have only one ship for each size